README.md

```diff
@@ -1,135 +1,131 @@
+# App Hosting adapters
 
 ## Overview
 
-Firebase Hosting integrates with popular modern web frameworks including Angular and Next.js. Using Firebase Hosting and
-Cloud Functions for Firebase with these frameworks, you can develop apps and microservices in your preferred framework
-environment, and then deploy them in a managed, secure server environment. Support during this early preview includes
-the following functionality:
+App Hosting provides no-config-needed build and deploy support for Web apps developed in these frameworks:
 
-* Deploy Web apps comprised of static web content
-* Deploy Web apps that use pre-rendering / Static Site Generation (SSG)
-* Deploy Web apps that use server-side Rendering (SSR)—full server rendering on demand
+* Next.js 13+
+* Angular 17.2+
 
-Firebase provides this functionality through the Firebase CLI. When initializing Hosting on the command line, you
-provide information about your new or existing Web project, and the CLI sets up the right resources for your chosen Web
-framework.
+This repo holds the code for the adapters that enable support for these frameworks. At a high level these adapters transform frameowkr specific configurations into an output bundle spec that App Hosting is able to process and configure frameworks support. For more information see our [documentation](https://firebase.google.com/docs/app-hosting/about-app-hosting#frameworks)
 
-We'd love to learn from you. [Express your interest in helping us shape the future of Firebase Hosting here.](https://goo.gle/41enW5X)
+## App Hosting output bundle
 
-## Status
+The App Hosting output bundle is a file based specification that allows different frameworks to configure and customize their App Hosting deploy to provide more full featured support.
 
-![Status: Experimental](https://img.shields.io/badge/Status-Experimental-blue)
+Any framework that can generate a build output in accordance with the App Hosting output bundle can be deployed on App Hosting.
 
-This repository is maintained by Google but is not a supported Firebase product. Issues here are answered by
-maintainers and other community members on GitHub on a best-effort basis.
+The output bundle primarily consists of a bundle.yaml file that sits inside of the .apphosting directory. This bundle.yaml contains all the ways that frameworks can configure App Hosting when users deploy their applications.
 
-[Please open issues related to Web Frameworks support in Firease CLI in the firebase-tools repository](https://github.com/firebase/firebase-tools/issues/new/choose).
+NOTE: App Hosting technically supports all all node applications, but no custom framework features will be enabled without the output bundle.
 
-## Enable framework-awareness
+## Output bundle Schema
 
-An experimental add-on to the Firebase CLI provides web framework support. To enable it, call the following:
+The output bundle all lives inside a single file:
```

Comment:
Let's say "is contained in a single file"

Reply:
Changed!